File: Cli_Input_JavaFX_Test/src/cli_input_javafx_test/Cli_Input_Item.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package cli_input_javafx_test;

/**
 *
 * @author mike
 */
public class Cli_Input_Item {

    protected Input_Cmd_Type Type;
    protected String Text;
    
    Cli_Input_Item(Input_Cmd_Type type, String text) {
        Type = type;
        Text = text;
    }
    
    public Input_Cmd_Type Type_Get() {
        return Type;
    }
    
    public String Text_Get() {
        return Text;
    }

}

File: Cli_Core_CS/Cli_Core_CS/Cli_Input_Item.cs
using System;

namespace Cli_Core_CS
{
    class Cli_Input_Item
    {
        Input_Cmd_Type Type;
        String Text;

        public Cli_Input_Item(Input_Cmd_Type type, String text)
        {
            Type = type;
            Text = text;
        }

        public Input_Cmd_Type Type_Get()
        {
            return Type;
        }

        public String Text_Get()
        {
            return Text;
        }
    }
}
